website_portal_purchase/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Antiun Ingeniería S.L. - Sergio Teruel
+# (c) 2015 Antiun Ingeniería S.L. - Carlos Dauden
```

Comment:
ok. Well I think it was already discussed at the OCA, but why not. As for (c) instead of Copyright, I'm not sure if it's bad but at Akretion we use "Copyright" for all our contribs.